### PR TITLE
[feature1] First implementation of feature1

### LINKED ISSUE #5: request
new request

### PR DESCRIPTION
**Pull request by Oleg-Birukov**
Following changes were made:

- new-feature.txt file was created
- feature1 folder was added

### CODE REVIEW

File: feature1/another-file.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Another file for feature1 branch.
```
Comment (Curchu) [inline feature1/another-file.txt:1]: Good!

Comment (Curchu) [reply feature1/another-file.txt:1]: thank you

**Review by Curchu — APPROVED** [review]
Done!

**Comment by Oleg-Birukov** [comment]
Closed

**Review by Curchu — COMMENTED** [review]
dfdsf

**Pull request merged**